src/edit.js

```diff
@@ -118,7 +124,7 @@ export default function MapsBlockAppleEdit( props ) {
 			return;
 		}
 
-		map.current.addMarkers( markers );
+		debouncedUpdateMarkers( markers );
```

Comment:
Because we checked for `map.current` in the debounce call above, this can be shortened like this:

```
useEffect( () => debouncedUpdateMarkers( markers ), [ markers ] );
```